pallets/eth-bridge/src/lib.rs

```diff
@@ -884,26 +943,26 @@ impl<T: Config> Pallet<T> {
                     active_range.event_types_filter.into_iter().collect::<Vec<ValidEvents>>();
 
                 let decoded_hex =
-                hex::decode("418da8f85cfa851601f87634c6950491b6b8785a6445c8584f5658048d512cae").
-                expect("test"); 
+                    hex::decode("418da8f85cfa851601f87634c6950491b6b8785a6445c8584f5658048d512cae")
```

Comment:
I know this is existing code, but what does this represent? Shouldn't we be handing any errors correctly or emitting a more descriptive `expect` message?

Reply:
its a dummy implementation, see the TODO few lines bellow